### PR TITLE
Make DEBUGGING the default on CI for Windows

### PR DESCRIPTION
**Pull request by khwilliamson**
This should help find things earlier/easier

### CODE REVIEW

File: .github/workflows/testsuite.yml [hunk 1]
```diff
@@ -373,7 +373,51 @@ jobs:
...
-          nmake CCTYPE=MSVC142 test
+          nmake CCTYPE=MSVC142 CFG=Debug test
+
+  #===============================================
+
+  windows-msvc100:
+    name: "Windows msvc100"
+    runs-on: windows-latest
+    timeout-minutes: 120
+    needs: sanity_check
```
Comment (tonycoz) [inline .github/workflows/testsuite.yml]: Did you intend to reinstate the broken `msvc100` test entry here?

(I suspect a rebase of an old, old branch.)

**Review by tonycoz — CHANGES_REQUESTED** [review]

File: .github/workflows/testsuite.yml (hunk 1 above)
Comment (khwilliamson) [reply .github/workflows/testsuite.yml]: Indeed, your suspicion is correct, and I didn't pay attention.

File: .github/workflows/testsuite.yml [hunk 2]
```diff
@@ -373,7 +373,7 @@ jobs:
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           cd win32
-          nmake CCTYPE=MSVC142 test
+          nmake CCTYPE=MSVC142 CFG=Debug test
```
Comment (tonycoz) [inline .github/workflows/testsuite.yml:376]: The CFG=Debug is needed in the Build entry too, for both jobs.

Sorry I missed this the first time around.

**Comment by khwilliamson** [comment]
@tonycoz, hopefully fixed

File: .github/workflows/testsuite.yml [hunk 3]
```diff
@@ -362,7 +362,7 @@ jobs:
         run: |
           call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           cd win32
-          nmake CCTYPE=MSVC142
+          nmake CCTYPE=MSVC142 CFG=Debug test
```
Comment (tonycoz) [inline .github/workflows/testsuite.yml]: This is the build step, it doesn't need "test".

**Review by tonycoz — APPROVED** [review]

**Pull request merged**